.gitignore

```diff
@@ -1,5 +1,6 @@
 # GlueSQL project files
 /data/
```

Comment:
Can't you delete this?

Reply:
I checked but we may need to keep both because..
```rust
cargo run --example ..
```
in the root directory still works :D

Reply:
Ah, such a case... Hmm, I'd like to recommend using it in the root.
What about `**/data` if it's hard?

Reply:
:D That is also.. hard to consider, I think. because in `src/` directory `data` is commonly used as module names.

Reply:
Oh, that's right... It looks good now.